client/lib/analytics/utils/save-coupon-query-argument.js

```diff
@@ -17,8 +17,15 @@ export default function saveCouponQueryArgument() {
 	}
 
 	// Read coupon list from localStorage, create new if it's not there yet, refresh existing.
-	const couponsJson = window.localStorage.getItem( MARKETING_COUPONS_KEY );
-	const coupons = JSON.parse( couponsJson ) || {};
+	let coupons = null;
```

Comment:
This is minor, but this block of code (plus the accompanying comment) looks like a function in disguise to me.